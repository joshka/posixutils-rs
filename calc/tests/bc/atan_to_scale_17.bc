a = a(0.2)
b = a(0.6)
c = a(1)
d = a(1.4)
e = a(1.8)

scale = 17
a/1
b/1
c/1
d/1
d/1
e/1
quit
