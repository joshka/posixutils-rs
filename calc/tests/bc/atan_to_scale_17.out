0.19739555984988075
0.54041950027058415
0.78539816339744830
0.95054684081207514
0.95054684081207514
1.06369782240255966
